client/extensions/belgaCoverage/src/belga-coverage-images.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import {ISuperdesk} from 'superdesk-api';
+
+import {getCoverageImages, IBelgaImage} from './belga-image-api';
+
+interface IProps {
+    maxImages: number;
+    coverageId: string;
+    rendition: 'preview' | 'thumbnail';
+    superdesk: ISuperdesk;
+}
+
+interface IState {
+    loading: boolean;
+    images: Array<IBelgaImage>;
+}
+
+function getImageUrl(image: IBelgaImage, rendition: IProps['rendition']): string {
+    switch (rendition) {
+        case 'preview':
+            return image.previewUrl;
+        case 'thumbnail':
+            return image.thumbnailUrl;
+    }
+}
+
+export default class BelgaCoverage extends React.Component<IProps, IState> {
+
+    readonly state = {loading: false, images: [] as IBelgaImage[]};
```

Comment:
Replace class property initializer with a constructor and TypeScript will infer the type from the state signature correctly without having to use `as` syntax which should be avoided when possible.